config/routes.rb

```diff
@@ -74,8 +77,6 @@
       post '/' => 'users/sessions#create', as: :user_session
       get '/logout' => 'users/sessions#destroy', as: :destroy_user_session
       delete '/logout' => 'users/sessions#destroy'
-      get '/active' => 'users/sessions#active'
```

Comment:
Would we need to keep the current routes for at least one deploy for backwards compatibility?

Reply:
> Would we need to keep the current routes for at least one deploy for backwards compatibility?

Ooh, yes, good call on that. That might make things complicated, since I expect we may need to keep a fair bit of the old logic around.

Reply:
> Would we need to keep the current routes for at least one deploy for backwards compatibility?

I think I captured all of what we'll need to keep around in cc4a481.